src/doc/rustc/src/platform-support.md

```diff
@@ -207,6 +207,7 @@ target | std | host | notes
 -------|:---:|:----:|-------
 `aarch64-apple-ios-macabi` | ? |  | Apple Catalyst on ARM64
 `aarch64-apple-tvos` | * |  | ARM64 tvOS
+`aarch64-apple-watchos-sim` | ✓ |  | ARM64 Apple WatchOS Simulator
```

Comment:
Please add a `platform-support/` file describing the target, the maintainers, etc. There's a template you can use here: https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support/TEMPLATE.md

I think it is largely fine if you only add just one file for all of the watchos targets and link to it from here.

Reply:
added